test/test_components.py

```diff
@@ -1474,10 +1474,9 @@ def test_component_functions(self):
 
 class TestNames:
     # This test ensures that `components.get_component_instance()` works correctly when instantiating from components
-    def test_no_duplicate_uncased_names(self):
-        subclasses = gr.components.Component.__subclasses__()
-        unique_subclasses_uncased = {s.__name__.lower() for s in subclasses}
-        assert len(subclasses) == len(unique_subclasses_uncased)
+    def test_no_duplicate_uncased_names(self, io_components):
```

Comment:
Unrelated, but I realized that this test was set up incorrectly since it only looked at the 4 direct subclasses of `gr.Component` instead of all of the descendants. Refactored to use the fixture.